.github/workflows/main.yaml

```diff
@@ -21,7 +22,7 @@ jobs:
     - name: Get UCX
       run: |
         git clone ${OPENUCX_LINK} /tmp/ucx
-        cd /tmp/ucx &&
+        cd /tmp/ucx && git checkout fd3e116e7f9ff61133918e1b06076d5d31f1ba6c
```

Comment:
Looks like switch to older version UCX make all tests passed (except Alltoall due to recent API change, see #21 ) (cc @bureddy)